### ISSUE TITLE
Next Branch Vue.use is not a function

### ISSUE THREAD

**Issue by 19peaches** [tochoromero/vuejs-smart-table#61]
I started a feature branch in my app using this package a while back before you updated it to support demi and vue 3.

I followed the directions by updating to the next branch, installing demi and the composition api package.

It installed fine and when a built it, no issues.

But when hitting a page with a smart table I get the following in my chrome inspector:

```
Uncaught TypeError: __webpack_provided_window_dot_Vue.use is not a function
    at Module../node_modules/@vue/composition-api/dist/vue-composition-api.esm.js (app.js:1902)
    at __webpack_require__ (manifest.js:23)
    at Module../node_modules/vuejs-smart-table/node_modules/vue-demi/lib/index.esm.js (app.js:101631)
    at __webpack_require__ (manifest.js:23)
    at Module../node_modules/vuejs-smart-table/dist/vuejs-smart-table.es.js (app.js:101563)
    at __webpack_require__ (manifest.js:23)
    at Module../resources/js/vuejs.js (app.js:28449)
    at __webpack_require__ (manifest.js:23)
    at Object../resources/js/app.js (app.js:20289)
    at __webpack_require__ (manifest.js:23)
```

Using this in Laravel 8 with Laravel-Mix 6 and Webpack 5.

Thanks

**Comment by tochoromero** [comment]
> installing demi and the composition api package
Hello, you don't need to install demi since that is a dependency of the package already, and you don't need to install the composition api package unless you are using Vue 2.

**Comment by 19peaches** [comment]
I am using Vue 2, that's the point here.

Installing demi would simply dupe it, it shouldn't cause an issue, certainly not the one I reported.

**Comment by tochoromero** [comment]
Yesterday I fixed some bugs I had with regards to Vue 2.
I just tested on a fresh Vue 2 CLI installation and it worked as expected.

Would you please update to the latest version and try again.

**Comment by tochoromero** [comment]
@19peaches where you able to make it work?

**Comment by 19peaches** [comment]
No. This is used in a live site I had to revert to 0.8 or whatever the previous was. I don't have time to fiddle around with Vue 3 nonsense.

**Comment by 19peaches** [comment]
I no longer see the releases/tags for anything greater than 0.0.7

Have you removed them?

**Comment by tochoromero** [comment]
No, I still see version 0.0.8 as `latest`.

<img width="791" alt="Screen Shot 2021-06-22 at 5 30 25 AM" src="https://user-images.githubusercontent.com/4211135/122924797-01a37780-d31b-11eb-9598-964bfae615cb.png">
